spread.yaml

```diff
@@ -1430,6 +1432,10 @@ suites:
             "$TESTSLIB"/prepare-restore.sh --prepare-suite
         prepare-each: |
             "$TESTSLIB"/prepare-restore.sh --prepare-suite-each
+            # When snapd.deb is from the repo, we just need to test when re-exec is enabled
+            if [ "$SNAP_REEXEC" = "0" ] && tests.info is-snapd-pkg-repo; then
+                touch .skip
```

Comment:
hmm `.skip` won't show up by default if you're in a debug shell. Can we use something more explicit? eg `tests.skip-stamp` and record the reason for skipping eg.:
```
echo "skipped due to snapd package installed from the distro's repository" > tests.skip-stamp
```

FWIW, it'd be amazing if there was a way to skip tasks in spread, such that a message is recorded in its output.

Reply:
updated

Reply:
Yeah, but we need to wait for new spread maintainer to start making changes and accepting proposals. I think this is great for now. The chief advantage is that we skip without duplicating the condition.